static/app/views/settings/account/apiApplications/details.tsx

```diff
@@ -22,6 +26,30 @@ type State = {
 } & DeprecatedAsyncView['state'];
 
 class ApiApplicationsDetails extends DeprecatedAsyncView<Props, State> {
+  rotateClientSecret = async () => {
+    try {
+      const rotateResponse = await this.api.requestPromise(
+        `/api-applications/${this.props.params.appId}/rotate-secret/`,
+        {
+          method: 'POST',
+          data: {},
+        }
+      );
+      openModal(({Body, Header}) => (
+        <Fragment>
+          <Header>{t('New Client Secret (ONE TIME ONLY)')}</Header>
```

Comment:
Using capitalization like this will cause this modal to feel a little bit "amateur". I think what you really would want to do here is to use an [`Alert`](https://github.com/getsentry/sentry/blob/8a85e901e085e2e3b0d9a04cd858348bdddfeb6a/static/app/components/alert.tsx) in the body that says something like

> This will be the only time your client secret is visible!

Then just have the header say `Rotated Client Secret`.

Reply:
A better way to think about this might be with our [Writing Style Guide](https://www.notion.so/sentry/Writing-Style-Guide-5b54fc27656c4003bcec4802effcf1eb?pvs=4) in mind

all caps creates a demanding, prescriptive tone which we want to avoid

(thanks @vuluongj20 for the words here!)